content/automate/ManageIQ/Transformation/Common.class/__methods__/assesstransformation.rb

```diff
@@ -25,6 +25,7 @@ def populate_factory_config
 
           def main
             raise 'Preflight check has failed' unless @task.preflight_check
+            @task.set_option(:network_mappings, @task.network_mappings)
```

Comment:
@fdupont-redhat Seems odd to take the method data from a task and put it "into the task options hash for later consumption".  Is this so it can be used through the API later?  If so it would be better to expose it there then to store the data that the object already generates.

cc @roliveri @agrare

Reply:
It's a workaround until I find why it's not set by the call to `preflight_check`. Without it, migration fails and blocks QE.

Reply:
@fdupont-redhat I need to understand what this really means:
> why it's not set by the call to preflight_check` 

I suggest we focus on this a little before jumping right to this work-around.  We can discuss.